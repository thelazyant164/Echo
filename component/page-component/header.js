import React from 'react';
import { StyleSheet, Text, View } from 'react-native';


export const Header = () =>
{
    return (
    <View>
        <Text>Echo</Text>
    </View>
    )
}
