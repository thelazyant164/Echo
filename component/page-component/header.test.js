import { React } from 'react';
import renderer from 'react-test-renderer';
import { Header } from './header';

describe('<Header />', () => {
  let header;
  beforeEach(() => {
    header = renderer.create(<Header />).toJSON();
  });

  test('Renders correctly', () => {
    expect(header).toMatchSnapshot();
  });
});
