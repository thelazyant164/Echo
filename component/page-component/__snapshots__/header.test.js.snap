// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#D9D9D9",
      "height": 60,
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 24,
        "justifyContent": "center",
        "marginLeft": 20,
        "marginTop": 10,
      }
    }
  >
    Echo
  </Text>
</View>
`;
